### PR TITLE
Issue #133 resolved : Added college in contestants.json

### LINKED ISSUE #133: Add a field for college in contestants.json and display it
add field for college in contestants.json and display college name in the about section in the card. 
update the steps in the readme too.

### PR DESCRIPTION
**Pull request by aayushjoglekar**
Live Preview not available as contestant.js is fetching data from opencode18 repo. Included Screenshot. 

Will work when contestants.json is merged.

1. Included college key in contestants.json
2. Included update about college in readme.

image

closes #133

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -52,14 +55,17 @@
          "name":"Madhur Gupta",
          "imageurl":"contestant/myimage.jpg",
          "about":"A wanna-be Web Developer.",
+         "college":"",
          "facebook":"https://www.facebook.com/madhurguptaiiita",
          "github":"https://github.com/madhur1998",
          "twitter":""
       },
+      "college" : "",
```
Comment (pradeepgangwar) [inline contestants.json]: Remove this

**Comment by pradeepgangwar** [comment]
@aayushjoglekar If you are on git. Try squashing commits. It's okay if you are not able to squash, just comment here.

**Comment by aayushjoglekar** [comment]
@pradeepgangwar Updated the json file.

**Comment by pradeepgangwar** [comment]
@aayushjoglekar README looks broken. Edit it and see it in preview.

**Comment by aayushjoglekar** [comment]
@pradeepgangwar  README is fixed.

**Comment by pradeepgangwar** [comment]
Learn git and about squashing commits, will be useful for future.

**Pull request merged**